docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,52 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main feature of the Metadata API is extraction of a metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.
```

Comment:
@pkosko Just to maintain writing style between pages, your sentence was also correct. 
```suggestion
The main feature of the Metadata API is to extract metadata from a file. You can [extract metadata](#extract-metadata) from multimedia files using the `get()` method.